tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/Matter/MatterRepository.cs

```diff
@@ -1073,6 +1073,7 @@ public bool SetItemPermission(ClientContext clientContext, IList<IList<string>>
 
                     if (null != roleCollection && 0 < roleCollection.Count && 0 < usersToRemove.Count)
                     {
+                        usersToRemove = usersToRemove.Where(user => user != string.Empty).ToList();
```

Comment:
@premchandpl add validation at client side to ensure empty records are not getting added

---
In reply to: [107745414](https://github.com/Microsoft/mattercenter/pull/703#discussion_r107745414) [](ancestors = 107745414)